### DISCUSSION TITLE
add nonce check to jwtVerify

### DISCUSSION THREAD

**Discussion by abbaspour** [panva/jose#302] [Ideas]
I think nonce is an import claim currently missing from `JWTVerifyOptions` in `JWTClaimVerificationOptions` type here: https://github.com/panva/jose/blob/main/src/types.d.ts#L502-L544

simple workaround is verify nonce against payload.nonce after jwtVerify, but built-in support for nonce is welcomed.

**Comment by panva** [comment]
I don't feel like bloating the current interface with JWT profile-specific claim validations or profiles in general. 

I highly welcome anyone to wrap around jose just exposing profiled jwt validators.
